content/collections/experiment_troubleshooting/en/new-experiment-run.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 71dedb3e-eb17-4ed1-9df0-bcbc78a2e6a2
+blueprint: experiment_troubleshooting
+title: 'New Experiment Run'
+author: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+source: 'https://www.docs.developers.amplitude.com/experiment/guides/troubleshooting/restarting-experiments/'
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1717179968
+---
+Creating a new run of an existing experiment can be useful if you had instrumentation issues that affected data quality, and you've since fixed them. When you create a new run, you exclude any previous user data from the monitoring and analysis of your experiment.
+
+## Prerequisites
```

Comment:
This section no longer applies but some of the info can be added to the experiment key section below.

The description can be something like: 
> By default, your experiment runs are delimited by time but you can optionally differentiate your experiment runs using the experiment key tracked on the default exposure event to ensure your new run does not include stale evaluated users. After you have created your new experiment run, enable the setting under the exposure event control to use experiment key. 

> There are two requirements to use experiment key:

Then you can keep the bullet points and table